tests/components/landing/ScrollIndicator.test.tsx

```diff
@@ -0,0 +1,28 @@
+import { render, fireEvent } from '@testing-library/react';
+import { ScrollIndicator } from '../../../src/components/landing/ScrollIndicator';
+import { describe, it, expect, vi } from 'vitest';
+
+describe('ScrollIndicator', () => {
+  it('renders with correct attributes', () => {
+    const { getByTestId } = render(<ScrollIndicator />);
+    expect(getByTestId('scroll-indicator')).toBeInTheDocument();
+    expect(getByTestId('scroll-arrow')).toBeInTheDocument();
+  });
+
+  it('scrolls when clicked', () => {
+    const scrollIntoViewMock = vi.fn();
+
+    const mockElement = document.createElement('div');
+    mockElement.id = 'what-is-section';
```

Comment:
no test cases for when the target element (`what-is-section`) is missing. this might lead to runtime issues.

Reply:
sure makes sense